src/surveys/components/Response.tsx

```diff
@@ -102,7 +104,6 @@ export default function Response({
     case AnswerType.CSAT:
       element = <div>CSAT</div>;
       break;
-    case AnswerType.MULTIPLE:
```

Comment:
@AfeezGL Why did you remove this? It's needed to rended `Multiple` response type.